Add parents condition (#35)
p2pcommons/specs: module.md

`p2pcommons.parents` (specific to `p2pcommons.type: content`) MUST be
an array of strings containing unique versioned Hyperdrive keys. These
versioned Hyperdrive keys MUST each refer to a valid module of
`p2pcommons.type: content` and MUST be registered by at least one of 

> its listed authors profile at the time this value is set. The network state SHOULD be up-to-date before determining registration status.

Reviewer: ```suggestion
its listed authors at the time this value is set. The network state 
SHOULD be up-to-date before determining registration status.
```

or 

```suggestion
its listed authors' profiles at the time this value is set. The network state
SHOULD be up-to-date before determining registration status.
```

Reviewer 2: 👍

`p2pcommons.parents` (specific to `p2pcommons.type: content`) MUST be
an array of strings containing unique versioned Hyperdrive keys. These
versioned Hyperdrive keys SHOULD each refer to a valid module of

> `p2pcommons.type: content` and MUST be registered by at least one of 
> its listed authors. If it contains this module's own Hyperdrive

Reviewer: I think this is a good question. To me it suffices that the content is registered at the version you currently have. It's not about preventing people from "tricking" the system and more about defining the process -> guiding people towards correct behaviour and clear user interactions imo.

```suggestion
`p2pcommons.type: content` and MUST be registered by at least one of 
its listed authors at latest known versions. If it contains this module's own Hyperdrive
```

> its self-designated authors. If it contains this module's own Hyperdrive

Reviewer: ```suggestion
its listed authors. If it contains this module's own Hyperdrive
```
I had to think for a moment about what "self-designated" meant. Perhaps "listed" is simpler. Otherwise great! :)

Reviewer 2: What does "registered" mean in a living system? I guess it must be registered to the author in the latest version of their profile known to the person validating it?
